postprocessor/src/main/java/com/azure/autorest/postprocessor/Postprocessor.java

```diff
@@ -131,6 +141,7 @@ public boolean processInternal() {
     }
 
     private void writeToFiles(Map<String, String> fileContents) throws FormatterException {
+        handlePartialUpdate(fileContents);
```

Comment:
You might add a setting to control whether to enable the feature.

Reply:
working on adding a flag to enable partial update feature